modules/FIC_Github.py

```diff
@@ -159,13 +159,13 @@ def _compare_versions(self):
             return False
 
     def _last_checked(self):
-        self.last_check = json.load(self.load(CHANGELOG_REPO_PATH, self.repo_name.lower() + ".json")).get("0").get("last_checked")
+        import datetime
+        self.last_check = datetime.datetime.strptime(json.load(self.load(CHANGELOG_REPO_PATH, self.repo_name.lower() + ".json")).get("0").get("last_checked"), "%Y-%m-%d %H:%M:%S")
 
```

Comment:
with 'def _last_check' method we only want to get the datetime string from .json files. 
```python
  {
  "0": {
"lastChecked": "2019-05-08 09:45:13.863194", )
```
and use it to get the commits since that time:
```python
self.repo_data.commits(since=self.last_check)
```
it's not required anything related with 'datetime'